Orange/widgets/data/owimpute.py

```diff
@@ -267,6 +303,11 @@ def create_imputer(self, method, *args):
             m = AsDefault()
             m.method = default
             return m
+        elif method == Method.Default and not args:  # global default values
+            return impute.FixedValueByType(
+                default_continuous=float(self.default_numeric or np.nan),
```

Comment:
`self.default_numeric` can be 0 which evaluates to False so np.nan is used instead!

Reply:
Oops. Fixed. Thanks.